apps/web/src/screens/HomePage/sections/Events/index.tsx

```diff
@@ -145,38 +171,60 @@ const CalendarLink = styled.a`
 `;
 
 const FILTER_TYPES: string[] = [
-  "ACM Comp",
-  "ACM Data",
-  "ACM Game",
-  "ACM General",
-  "ACM Hack",
-  "ACM Sec",
-  "ACM-W"
+  "Competition",
+  "Data",
+  "Game",
+  "General",
+  "Hack",
+  "Security",
+  "Women"
 ];
 
 const DEFAULT_EVENTS_TO_DISPLAY: number = 3;
 const CALENDAR_LINK: string =
   "https://calendar.google.com/calendar/embed?src=mst.edu_7u3stm8bn7l2umuastep5fmbl0%40group.calendar.google.com&ctz=America%2FChicago";
 
 const Events: React.FC<{}> = (): JSX.Element => {
-  const [filters, setFilters] = useState<boolean[]>([]);
+  const result: GetCurrentEventsQueryHookResult = useGetCurrentEventsQuery();
+  let events: any;
+  if (result.data && result.data.currentEvents) {
+    events = result.data.currentEvents;
+    events.sort(
+      (a: any, b: any) =>
+        new Date(a.dateHosted).getTime() - new Date(b.dateHosted).getTime()
+    );
+  }
+
+  const [filters, setFilters] = useState<boolean[]>([
```

Comment:
Try `new Array(FILTER_TYPES.length).fill(false);`. We can then generate this list via graphql dynamically later